Add snap package manager support to data provider (#15740) (#6084)
wazuh/wazuh-documentation: source/user-manual/capabilities/system-inventory/available-inventory-fields.rst

Packages
--------

> The ``sys_programs`` table in the inventory database stores information about the currently installed software on an endpoint. The Vulnerability Detector module uses information from this table to scan and detect vulnerable software. On Linux systems, retrieved packages can be *deb, pacman, rpm or snap*. The table below describes the fields in the database.

Reviewer: ```suggestion
The ``sys_programs`` table in the inventory database stores information about the currently installed software on an endpoint. The Vulnerability Detector module uses information from this table to scan and detect vulnerable software. On Linux systems, retrieved packages are *deb, pacman, rpm, and snap*. The table below describes the fields in the database.
```

+------------------+-------------------------------------------+-------------------------------------------+----------------------------------------------+
| ``multiarch``    | Multiarchitecture support                 | same                                      | Linux (deb)                                  |
+------------------+-------------------------------------------+-------------------------------------------+----------------------------------------------+
| ``source``       | Source of the package                     | linux-meta                                | Linux (deb/rpm/snap) and  macOS (pkg)        |
+------------------+-------------------------------------------+-------------------------------------------+----------------------------------------------+
| ``description``  | Description of the package                | Generic Linux kernel headers              | Linux (deb/rpm/pacman/snap) and macOS (pkg)  |
+------------------+-------------------------------------------+-------------------------------------------+----------------------------------------------+

> | ``location``     | Location of the package                   | C:\\Program Files\\VMware\\VMware Tools\\ | Linux (snap), Windows and  macOS (pkg)       |

Reviewer: ```suggestion
| ``location``     | Location of the package                   | C:\\Program Files\\VMware\\VMware Tools\\ | Linux (snap), Windows, and  macOS (pkg)      |
```
